autogpt/commands/file_operations.py

```diff
@@ -83,6 +84,24 @@ def split_file(
         start += max_length - overlap
 
 
+@command("read_text_file", "Read file", '"filename": "<filename>"')
```

Comment:
The command description might confuse the AI, since there is an another one with the same description.

I think “Read Text File” would be better. (Which might be still confusing) Maybe parse_text_document would be better?

Reply:
I agree with the provided command "parse_text_document". Should I continue adding this in the main repo as it is a very useful command or add it as a plugin?